index.js

```diff
@@ -6,11 +6,15 @@ var YAML = require('yamljs');
 /**
  *
  * @param {number|string} performanceYear - four digit year
- * @return {Array<Benchmark>} - benchmarks data for given year
+ * @return {Array<Benchmark>|undefined} - benchmarks data for given year
  */
 exports.getBenchmarksData = function(performanceYear) {
-  return JSON.parse(
-    fs.readFileSync(path.join(__dirname, 'benchmarks', performanceYear + '.json')));
+  try {
+    return JSON.parse(
+      fs.readFileSync(path.join(__dirname, 'benchmarks', performanceYear + '.json')));
+  } catch (error) {
+    return;
```

Comment:
Am I reading this correctly? 
Your first suggestion is to leave the try/catch out like it was before and just document the fact that that may happen and that anyone using this module should handle that on their end?

Your second suggestion is to add a build task that aggregates all of the historical benchmarks into one file (index file) that lives in the same place as this function so that we don't have to use `fs` in the first place.

Is that right? @kencheeto